pkg/reaper/secrets.go

```diff
@@ -92,3 +92,7 @@ func secretToEnvVars(secret *corev1.Secret, envUsernameParam, envPasswordParam s
 
 	return &usernameEnvVar, &passwordEnvVar, nil
 }
+
+func GetTruststoresSecretName(reaperName string) string {
```

Comment:
Suggestion: consider abbreviating this to -ts (like we do -ks for keystores) because otherwise the name will be too long for the API server.

Reply:
Pushed a commit doing this.